docs/content/users/configuration/config.md

```diff
@@ -393,6 +393,9 @@ Node.js version for the web container’s “system” version.
 
 There is no need to configure `nodejs_version` unless you want a version other than the default version.
 
+!!!tip "Select the Node.js version from a file such as `.nvmrc` or `package.json`"
+    `nodejs_version` can be set to `auto`, which will use `n install auto` under the hood.
```

Comment:
Probably worth either linking to `n` docs or explaining what that means. People who want this functionality may not know what `n` is or what `n install auto` does.

Reply:
Thanks, that's a good suggestion.